### ISSUE TITLE
Crash on opening a file in the filepool

### ISSUE THREAD

**Issue by dschreij** [smathot/OpenSesame#417]
When selecting a file from the open file dialog in the file pool, the following error occurs:

~~~
Information:

Operating System: darwin
OpenSesame: 3.1.0a36
Python: 3.5.1

Traceback (most recent call last):
  File "/Users/daniel/Github/OpenSesame/libqtopensesame/widgets/pool_widget.py", line 140, in select_and_add
    cfg.default_pool_folder = os.path.dirname(path_list[0])
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/posixpath.py", line 148, in dirname
    i = p.rfind(sep) + 1
AttributeError: 'list' object has no attribute 'rfind'
~~~

**Comment by dschreij** [comment]
This was caused due to an Qt4/Qt5 API inconsistency, fixed by #418

**Comment by smathot** [comment]
Thanks! I'll merge the fix.
